### ISSUE TITLE
Option precedence

### ISSUE THREAD

**Issue by jml9904** [axboe/fio#151]
Looks from options.c in option_dup_subs() that environment variables will override all other options. If options extant in the fio options file are specified on the command line, can/should they override whatever's in the command file? Doesn't appear that they do, and I would think the precedence would be command line [overrides] environment [overrides] option file.

Thanks...

**Comment by frukto** [comment]
Yes, it would be very nice if command line options override options given in job files.  I would like to do things like

`fio --runtime=30 jobs.fio` to ad-hoc modify parameters in otherwise carefully crafted job descriptions.

With `--showcmd` I see the options are just doubled and the ones from the file seems to have precedence (likely because they are given last).

NB: The linked PR seems to be totally unrelated
